lib/libc/semaphore/Make.defs

```diff
@@ -54,9 +54,7 @@
 
 CSRCS += sem_init.c sem_getprotocol.c sem_getvalue.c
 
-ifneq ($(CONFIG_PRIORITY_INHERITANCE),y)
```

Comment:
You should check this again.
There are two files with the same name for enabled and disabled the config.
It looks you don't understand the usages of them exactly.